package-lock.json

```diff
@@ -2378,12 +2384,6 @@
             "strip-ansi": "^5.0.0"
           }
         },
-        "y18n": {
-          "version": "4.0.1",
```

Comment:
De oude versie was niet 3.2.1 blijkbaar? Wat is deez?

Reply:
Dit id een dependency van `sass-graph` (kun je met een beetje moeite zien aan het verschil in indentatie).  Ik verwacht geen problemen dat dit hier weg is.